controllers/authorization/authorization_controller.go

```diff
@@ -32,10 +33,13 @@ func NewPlatformAuthorizationController(cli client.Client, log logr.Logger,
 			"controller", ctrlName,
 			"component", component.ObjectReference.Kind,
 		),
-		config:         config,
-		authComponent:  component,
-		typeDetector:   authorization.NewAnnotationAuthTypeDetector(metadata.Annotations.AuthEnabled),
-		hostExtractor:  authorization.NewExpressionHostExtractor(component.HostPaths),
+		config:        config,
+		authComponent: component,
+		typeDetector:  authorization.NewAnnotationAuthTypeDetector(metadata.Annotations.AuthEnabled),
+		// TODO: Evaluate passing in the hostExtractor to avoid coupling the authorizaiton/routing packages
```

Comment:
I agree with `TODO` - wondering if we shouldn't start converting those to some refactoring story in JIRA to they won't get lost.